entries/c/concur.com.json

```diff
@@ -1,10 +1,15 @@
 {
   "Concur": {
     "domain": "concur.com",
-    "contact": {
-      "facebook": "SAPConcur",
-      "twitter": "SAPConcur"
-    },
+    "tfa": [
+      "sms",
+      "call",
+      "totp",
+      "custom-software",
+      "u2f"
+    ],
+    "documentation": "https://dam.sap.com/mac/app/p/pdf/asset/preview/kA9GcJq?ltr=a&rc=10",
+    "notes": "2FA is only enabled on specific Concur instances by the sysadmin.  2FA must be enabled on instance between 18OCT2023-15NOV2023 initially."
```

Comment:
```suggestion
```

I don't think the notes are necessary.

I also don't see where the sysadmin can disable 2FA in the documentation you provided. On page two, it says "No, there is no option to disable the 2FA feature."